color.h

```diff
@@ -0,0 +1,23 @@
+#ifndef COLOR_H_
+#define COLOR_H_
+
+#include "majorandminor.h"  
+
+typedef struct {
+    MajorColor_tst majorColor;
+    MinorColor_tst minorColor;
+} ColorPair;
+
+
+#define COLORPAIRNAME_EXTRABUFFLEN      ((int)4)
```

Comment:
Nice names for calculating the buffer between colors when converting to string.
However, the COLORPAIRNAME_EXTRABUFFLEN is not used when actually converting to string. Could that lead to inconsistency when e.g., a maintainer changes the spacing?